package.json

```diff
@@ -1,43 +1,28 @@
 {
   "name": "create",
-  "version": "1.0.0",
+  "version": "3.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "test": "NODE_ENV=testing jest __tests__",
```

Comment:
Not sure we need to pass NODE_ENV, `jest` automatically sets it to `test`.

Reply:
we don't, removed.